ui/components/ui/toggle-button/index.scss

```diff
@@ -1,7 +1,7 @@
 .toggle-button {
   display: flex;
   cursor: pointer;
-
+  text-align: center;
```

Comment:
This file is used pretty globally, are we sure that this doesn't impact other toggles display?

Reply:
this is only used for <ToggleButton> component so this is what we wanted ~

Reply:
Also not sure we want to center the text for all instances of the `ToggleButton`? Which instance needs to be centered and can it conform to left aligned?

Reply:
Also not sure we want to center the text for all instances of the `ToggleButton`? Which instance needs to be centered and can it conform to left aligned?